tinywallet/tinywallet/screens.py

```diff
@@ -1786,7 +1786,9 @@ def onTicketItemClicked(self, item):
         """
         hostname = nets.normalizeName(cfg.netParams.Name)
         utxo = self.liveTickets[item.text()[:8]]
-        url = urlunsplit(("https", f"{hostname}.dcrdata.org", f"/{utxo.txid}", "", ""))
+        url = urlunsplit(
+            ("https", f"{hostname}.dcrdata.org/tx", f"/{utxo.txid}", "", "")
```

Comment:
Don't put the path in the `netloc` field, put it in the `path` field instead:

`("https", f"{hostname}.dcrdata.org", f"/tx/{utxo.txid}", "", "")`

Reply:
gah, that's what I did at first, I'm still so clueless when it comes to web